src/App.css

```diff
@@ -1,33 +1,17 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+.app {
+  margin: 20px 0 0 20px;
+  text-align: left;
+  color: #333333;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
+h1 {
+  margin: 20px auto;
+  text-align: center;
+  font-size: 20px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.restaurantList {
```

Comment:
There are a few methodologies that guide CSS development and classes naming convention. One of them is the [BEM - Block, Element, Modifier](http://getbem.com/introduction/). Further, you can read the [naming](http://getbem.com/naming/) page.

The best part about using both React and BEM is that, in React context, understanding what is a component and what is an element is a lot easier, as you usually do that when writing your own components.

As a plus for our project, I'd suggest you to adopt this methodology. It's not a hard methodology - you can learn it with a quick reading :pray: 

In summary, only your components' root element will be treated as a _Block_. All the other elements inside your component will be treated as an _Element_.